frontend/src/Components/AboutPage.js

```diff
@@ -9,65 +9,71 @@ class AboutPage extends React.Component {
-        <div class="container">
-          <h2 ><span>ACEKnow Intro</span></h2>
-          <p class="p1 text-center">
+        <div className="container">
+          <h2 className="h2-about"><span>ACEKnow Intro</span></h2>
+          <div className="contianer-about"><div className="front-break1"></div></div>
+          <p className="p1-about text-center">
           His title, King of Heroes, is not meant to call him a king who is a hero, but instead implies that he is the king over all heroes. The heroes of various myths are derived from his legend, so his Gate of Babylon possesses all of their Noble Phantasms. Though there are numerous kings with titles such as the King of Knights or the King of Conquerors, he is the only one in all of heaven and earth crowned with the title of "King of All Heroes."
           </p>
         </div>
 
+        <div className="contianer-about"><div className="end-break"></div></div>
+
         {/* ACE Intro */}
-        <div class="container">
-          <h2><span>ACE Intro</span></h2>
-          <br></br>
-          <div class="row align-items-center">
-            <div class="col-lg-6">
+        <div className="container">
+          <h2 className="h2-about"><span>ACE Intro</span></h2>
+
+          <div className="contianer"><div className="front-break"></div></div>
+
+          <div className="row align-items-center">
+            <div className="col-lg-6">
               <img
-              class="ace-img"
-              src="https://www.utace.club/static/userpage/images/aboutus.jpg"
+              className="ace-img"
+              src="https://media-exp1.licdn.com/dms/image/C561BAQFMxRUPohW0SA/company-background_10000/0/1554158299698?e=2159024400&v=beta&t=SC7lOclAPupM8ipIbylrAIpm7MTW5mBvIKG9ifiMM6w"
               alt="ACE" />
             </div>
-            <div class="col-lg-6">
-              <p class="p1">
+            <div className="col-lg-6">
+              <p className="p1-about">
                 Born with a body that was of the highest grade by mortal standards and knowledge reaching truth, Gilgamesh was born, designed, as king and the Wedge of Heaven to connect the rising humans and the fading gods. He was sent to ensure the humans and bind the earth slowly leaving the Age of Gods. He was a being embodying the two life sets of life forms, with the blood of those who had ruled and the blood of those who would rule from thereon.
               </p>
             </div>
           </div>
         </div>
 
+        <div className="contianer"><div className="end-break"></div></div>
+
         {/* Contact Us */}
-        <br></br><br></br>
-        <div class="container">
-          <h2><span>Contact Us</span></h2>
-          <div class="contact-box">
-            <div class="contact-row">
-              <div class="contact-column">
-                <div class="contact-card">
+        <div className="container">
+          <h2 className="h2-about"><span>Contact Us</span></h2>
```

Comment:
Also, why is there span inside of h2? It seems unecessary

Reply:
I use span to help build title styel (---ACE---)

Reply:
That makes sense. You can resolve this then.